specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -4265,7 +4396,11 @@
         },
         "vmCustomConfigurationUri": {
```

Comment:
The field names for custom configuration used in existing ARM template are customConfigurationTemplateUrl and customConfigurationParameterUrl.  I would like to suggest to use the same names in swagger?